mp/src/game/shared/momentum/mom_stickybomb.cpp

```diff
@@ -198,6 +198,10 @@ EXPOSE_INTERFACE(CStickybombArmTimeProxy, IMaterialProxy, "ArmTime" IMATERIAL_PR
 
 #else
 
+bool CMomStickybomb::GetUseImpactNormal() {
+    return m_bUseImpactNormal;
+}
```

Comment:
You will be removing this when tackling the other comment but as a future style note: braces should be on their own lines.